### DISCUSSION TITLE
An option to display files/directories as modified?

### DISCUSSION THREAD

**Discussion by nikola-vukicevic** [nvim-tree/nvim-tree.lua#1574] [Q&A]
Is there an option to clearly display that files open in buffers are modified (when they _are_ modified ofc), with a different color and a symbol, such as the Vim's default `[+]` (that can preferably be customized by the user)?

If not, I think that would be a reasonable feature request.

**Comment by alex-courtis** [comment]
When you say modified, are you referring to git dirty and other git statuses?

Modified buffers and their containing folders are shown by default.

**Comment by nikola-vukicevic** [comment]
I've edited the question in order to clarify what I am asking.

TLDR: I can't seem to find the option to have modified buffers visibly marked, preferably by the same `[+]` symbol Vim uses by default (or at least with a different color), so I'm wondering whether such an option even exists ATM, and if not, whether such an option can be added in the future.

Cheers

**Reply by alex-courtis** [reply]
Icons: `:help nvim-tree.renderer.icons.glyphs`

Colours: `help nvim-tree.renderer.highlight_git`

**Reply by nikola-vukicevic** [reply]
> Icons: `:help nvim-tree.renderer.icons.glyphs`
> 
> Colours: `help nvim-tree.renderer.highlight_git`

I don't think there is an answer to my original question in those two places (and I _have_ previously read those parts of the documentation).

There is an option to highlight open documents: `highlight_opened_files`, but I don't think I've seen an option to highlight **modified** documents, something along the lines of `"highlight_modified_files"`, and especially not an option to have an appropriate symbol (such as `[+]`) appended to the name of the file (in Nvim tree), which is opened in one of the buffers.

**Reply by alex-courtis** [reply]
I apologise: I did not properly re-read your question.

Functionality:
* highlight group for modified
* icon for modified
* folder modified as per git/lsp

**Reply by nikola-vukicevic** [reply]
Thank you

**Comment by alex-courtis** [comment]
Re-raised #1577
